tensorflow/lite/micro/kernels/cmsis-nn/depthwise_conv.cc

```diff
@@ -134,7 +133,6 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
     // Allocate memory for per-channel quantization parameters
     const int num_channels =
```

Comment:
num_channels here is not used and should be removed.

Reply:
Sorry, it's the other way around, the allocation should be moved here under (input->type == kTfLiteInt8). So the other num_channels below can be removed.